include/klee/Internal/Module/KInstruction.h

```diff
@@ -44,7 +44,8 @@ namespace klee {
 
   public:
     virtual ~KInstruction();
-    void printFileLine(llvm::raw_ostream &);
+    void printFileLine(llvm::raw_ostream &) const;
+    std::string printFileLine() const;
```

Comment:
I think `getLocation` sounds too different from the other function which is similar.  Let's keep it like this.